apache/incubator-zeppelin
# [ZEPPELIN-701] - Upgrade Tachyon Interpreter to Alluxio Interpreter

### What is this PR for?

Upgrading existing Tachyon interpreter to Alluxio interpreter.

### What type of PR is it?

improvement

### Is there a relevant Jira issue?

https://issues.apache.org/jira/browse/ZEPPELIN-701

### How should this be tested?

* [Install and configure Alluxio](http://www.alluxio.org/downloads/) on a local machine.
* run Alluxio in local mode ```$ ./bin/alluxio-start.sh local```
* check that interpreter params are setted to default values (hostname: localhost, port: 19998)
* use the [Alluxio CLI commands](http://www.alluxio.org/documentation/v1.0.0/en/Command-Line-Interface.html) to interact with your Alluxio file system

### Questions:

* Does the licenses files need update? no
* Is there breaking changes for older versions? no
* Does this needs documentation? no
